ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3600,6 +3600,21 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
   - EFSS2X
   - HT2ZC
   - CO2C
+  
+AHU_SFVSC_CHWRC_OAFC_SSPC_RAVOCC_RACO2C_ESFM:
```

Comment:
You need to have some form of fan SS included, as well.

Reply:
Added